src/OrleansAzureUtils/Providers/Streams/AzureQueue/AzureQueueBatchContainer.cs

```diff
@@ -102,6 +112,14 @@ public bool ImportRequestContext()
             return false;
         }
 
+        public void GetObjectData(SerializationInfo info, StreamingContext context)
```

Comment:
Don't you need the `EventSequenceToken sequenceToken` as well?
Not sure it will be captured now, and even if it will be, I think its better to use one way to serialize and not mix 2 ways.